server/routes/vizRouter.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import { IRouter, IKibanaResponse, ResponseError } from '../../../../src/core/server';
+import { API_PREFIX } from '../../common';
+import { RequestParams } from '@elastic/elasticsearch';
+
+export function vizRouter(router: IRouter) {
+  router.get(
+    {
+      path: `${API_PREFIX}/visualizations`,
+      validate: {},
+    },
+    async (context, request, response): Promise<IKibanaResponse<any | ResponseError>> => {
+      const params: RequestParams.Search = {
+        index: '.kibana',
```

Comment:
Does this request to search .kibana for viz? Why don't we create an extra index like `.notebooks`?

Reply:
This router is used only on UI to get saved objects from '.kibana' index that are available to the user as per their defined role. The once the user selects a saved object it is stored in notebook as a paragraph in '.notebooks' index by the backend.

Reply:
Ok sounds good